lhotse/recipes/mgb2.py

```diff
@@ -130,7 +130,7 @@ def prepare_mgb2(
             ) as f_out:
                 for line in f_in:
                     f_out.write(line.replace("wav/", f"{corpus_dir}/{part}/wav/"))
-                    f_out.write("\n")
```

Comment:
Wouldn’t the script write everything to a single line after this change?

Reply:
The lines read from the wav.scp file already contain a new line, so the additional "\n" adds empty line. However I agree that it will be cleaner to just filter out the empty lines from any files that is being read from kaldi folder.

Reply:
As Dan pointed out, Kaldi doesn’t allow empty lines, so let’s strive to produce correct outputs. Please fix style issues and LGTM.

Reply:
The new line was coming from the line `f_out.write("\n")` which I removed now. I tested it and now it works perfectly with load_kaldi_data_dir().

Reply:
Cool. Please fix black issues and LGTM.

Reply:
Done